backend/src/app/rpc/commands/auth.clj

```diff
@@ -375,6 +377,8 @@
                                              (not (contains? cf/flags :email-verification)))
                                params    (-> params
                                              (assoc :is-active is-active)
+                                             (assoc :newsletter-updates accept-newsletter-updates)
+                                             (assoc :newsletter-news accept-newsletter-news)
```

Comment:
With the workflow change on the UI side, I think we need respect the backend API flow, so all this additional parameters can be and I think should be handled on the prepare-register.